doorbot.js

```diff
@@ -2,10 +2,12 @@ const https = require('https');
 const parse = require('url').parse;
 const format = require('url').format;
 const stringify = require('querystring').stringify;
+const crypto = require("crypto");
 
 const logger = require('debug')('doorbot');
 
 const API_VERSION = 9;
+const hardware_id = crypto.randomBytes(16).toString("hex");
```

Comment:
Why is this needed? From my tests the value of this doesn't matter..

Reply:
The floodlights would always return a 401 to the PUT endpoint, until I changed the hardware Id. Also, I was getting a bunch of retries before the change, then no retries afterwards.

Honestly man, I think ring added a check for that hardware Id and they are tampering with the results.

Reply:
Could be, they didn't like that I made this since they didn't offer a public API when people clearly want it :)

Reply:
Yea man, it sucks that we have to circumvent like this. Thanks for the merge